.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
 /lib
+.DS_Store
```

Comment:
We shouldn't add ignoring files generated by OS, editor, IDE and so on.
Because we don't want to add `.DS_Store` to all repo's .gitignore.

https://git-scm.com/docs/gitignore

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.